rfcs/0002-branch-generation.md

```diff
@@ -0,0 +1,76 @@
+# Branch Generation
+
+The goal of this RFC is to define how Solana generates branches.
+
+
+## Basic Design Idea
+
+Nodes take turns being leader and generating the PoH that encodes state changes.  The network can tolerate loss of connection to any leader by synthesizing what the leader ***would have generated*** had it been connected but not ingesting any state changes.  The complexity of forks is thereby limited to a "there/not-there" skip list of branches that may arise on leader rotation periods boundaries.  A leader can only transmit durring their predefined PoH slot.
```

Comment:
"complexity of forks .. list of branches"  huh?

on a boundary or after?

A leader can transmit at any time, but it's transactions will only be accepted if they fall within the leader's slot.